server/src/completions/variable-flags.ts

```diff
@@ -0,0 +1,12 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) 2023 Savoir-faire Linux. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+export const VARIABLE_FLAGS = [
```

Comment:
How these have been selected? I see a quite different set of bitbake defined variable flags in the doc: https://docs.yoctoproject.org/bitbake/bitbake-user-manual/bitbake-user-manual-metadata.html#variable-flags

Reply:
I doubt this one there has everything. Because I know there is a flag called `doc` but it is not listed.

Reply:
Let's bring this along with the topic of fetching docs from remote to the client sync.